### PR TITLE
no more into

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin @sdroege 

It's very likely that I missed some. Future will tell us. :)

### CODE REVIEW

**Comment by EPashkin** [comment]
```
error[E0308]: mismatched types
  --> src\bin\builder_basics.rs:21:28
   |
21 |     window.set_application(application);
   |                            ^^^^^^^^^^^
   |                            |
   |                            expected enum `std::option::Option`, found reference
   |                            help: try using a variant of the expected type: `Some(application)`
   |
   = note: expected type `std::option::Option<&_>`
              found type `&gtk::Application`

error[E0308]: mismatched types
  --> src\bin\menu_bar_system.rs:43:17
   |
43 |     menu.append("Quit", "app.quit");
   |                 ^^^^^^
   |                 |
   |                 expected enum `std::option::Option`, found reference
   |                 help: try using a variant of the expected type: `Some("Quit")`
   |
   = note: expected type `std::option::Option<&str>`
              found type `&'static str`

error[E0308]: mismatched types
  --> src\bin\notebook.rs:38:58
   |
38 |         let close_image = gtk::Image::new_from_icon_name("window-close",
   |                                                          ^^^^^^^^^^^^^^
   |                                                          |
   |                                                          expected enum `std::option::Option`, found reference
   |                                                          help: try using a variant of the expected type: `Some("window-close")`
   |
   = note: expected type `std::option::Option<&str>`
              found type `&'static str`
```

File: src/bin/accessibility.rs [hunk 1]
```diff
@@ -17,16 +17,16 @@ use std::env::args;
...
-    let button = gtk::Button::new_with_label("Click me!");
-    let label = gtk::Label::new("0");
+    let button = gtk::Button::new_with_label(Some("Click me!"));
+    let label = gtk::Label::new(Some("0"));
     let vbox = gtk::Box::new(gtk::Orientation::Vertical, 0);
 
     if let (Some(button_obj), Some(label_obj)) = (button.get_accessible(), label.get_accessible()) {
         // We set the description
-        button_obj.set_description("Button to increase label value");
+        button_obj.set_description(Some("Button to increase label value"));
```
Comment (EPashkin) [inline src/bin/accessibility.rs]: this unneeded

File: src/bin/accessibility.rs [hunk 2]
```diff
@@ -17,16 +17,16 @@ use std::env::args;
 fn build_ui(application: &gtk::Application) {
     let window = gtk::ApplicationWindow::new(application);
 
-    window.set_title("Accessibility");
+    window.set_title(Some("Accessibility"));
     window.set_position(gtk::WindowPosition::Center);
 
-    let button = gtk::Button::new_with_label("Click me!");
-    let label = gtk::Label::new("0");
+    let button = gtk::Button::new_with_label(Some("Click me!"));
```
Comment (EPashkin) [inline src/bin/accessibility.rs]: this too

**Comment by sdroege** [comment]
Good for me once it compiles

**Pull request merged**